<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profInsertionForm</class>
 <widget class="QWidget" name="profInsertionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="profNameFrame">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>90</y>
     <width>235</width>
     <height>45</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QLabel" name="profNameLabel">
      <property name="text">
       <string>نام استاد</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="profName"/>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="profGradeFrame">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>179</width>
     <height>45</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QComboBox" name="profGrade">
      <item>
       <property name="text">
        <string>مربی آموزشیار (لیسانس)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>مربی آموزشیار (فوق لیسانس)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>مربی (فوق لیسانس)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>مربی (دکتر)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>استادیار</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>دانشیار</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>استاد</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="profGradeLabel">
      <property name="text">
       <string>مرتبه استاد</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="profBaseFrame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>179</width>
     <height>45</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QComboBox" name="profBase">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>24</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>26</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>27</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>28</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>29</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="profBaseLabel">
      <property name="text">
       <string>پایه استاد</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="scienceCommitteeFrame">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>170</y>
     <width>179</width>
     <height>45</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QComboBox" name="scienceCommittee"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="scienceCommitteeLabel">
      <property name="text">
       <string>هیئت علمی</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="rightToTeachFrame">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>179</width>
     <height>45</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QComboBox" name="rightToTeach"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="rightToTeachLabel">
      <property name="text">
       <string>حق التدریس</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="appendProfBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>افزودن استاد</string>
   </property>
  </widget>
  <widget class="QFrame" name="profListSelectionFrame">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>601</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QPushButton" name="profListExploreBtn">
      <property name="text">
       <string>جستجو</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="profListPath"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="profListSelectionLabel">
      <property name="text">
       <string>فایل اساتید</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
